fuel-tx/src/transaction/consensus_parameters.rs

```diff
@@ -273,6 +332,63 @@ impl From<ConsensusParametersV1> for ConsensusParameters {
     }
 }
 
+/// A collection of parameters for convenience
```

Comment:
It would be nice to mention what is the difference between V1 and V2 here=) You can check how we do it for GasCosts

Reply:
Update in https://github.com/FuelLabs/fuel-vm/pull/821/commits/65087ba3eb90fc5837ad621affde74f2fa1a7b51